src/software/ai/ai.cpp

```diff
@@ -4,9 +4,11 @@
 
 #include "software/ai/hl/stp/play/halt_play.h"
 #include "software/ai/hl/stp/stp.h"
+#include "software/ai/navigator/path_planner/theta_star_path_planner.h"
 
 AI::AI()
-    : navigator(std::make_shared<Navigator>()),
+    : navigator(std::make_shared<Navigator>(
+          std::make_unique<ThetaStarPathPlanner>(ThetaStarPathPlanner()))),
```

Comment:
Is the additional `ThetaStartPathPlanner()` intentional? If not I think u can just do `std::make_unique<ThetaStarPathPlanner>()` and avoid constructing and then calling the copy-constructor

Reply:
hm ok